website/docs/r/datacenter.html.markdown

```diff
@@ -46,6 +46,12 @@ The following arguments are supported:
 ~> **NOTE:** Tagging support is unsupported on direct ESXi connections and
 requires vCenter 6.0 or higher.
 
+* `custom_attributes` - (Optional) Map of custom attribute ids to value 
```

Comment:
Can we have a link to directions on how to apply custom attributes similar to what we have for tags [here](https://www.terraform.io/docs/providers/vsphere/r/tag.html#using-tags-in-a-supported-resource)?